cluster-agent/controllers/managed-gitops/eventloop/operation_event_loop_test.go

```diff
@@ -1196,6 +1198,32 @@ var _ = Describe("Operation Controller", func() {
 				err = k8sClient.Update(ctx, applicationCR)
 				Expect(err).To(BeNil())
 			}
+			refreshHandler := func(appCR appv1.Application, closeRefreshHandler chan struct{}) {
```

Comment:
Can you add a quick comment to this function? It's obvious what the purpose is in the context of this PR, but may not be obvious to a casual observer reading through the test after-the-fact.